xbmc/rendering/gl/RenderSystemGL.cpp

```diff
@@ -345,8 +345,7 @@ void CRenderSystemGL::CaptureStateBlock()
   glMatrixTexture.Push();
 
   glDisable(GL_SCISSOR_TEST); // fixes FBO corruption on Macs
-  if (glActiveTextureARB)
```

Comment:
LinuxRenderGL.cpp uses it without even asking - so it most likely had failed before if we once would have run into it.